package.json

```
@@ -18,10 +18,10 @@
   "private": true,
   "scripts": {
     "build": "run-p build:backend build:frontend copy",
-    "build:backend": "tsc -p tsconfig.backend.json && npm run build:backend:webpack",
+    "build:backend": "tsc -p tsconfig.backend.json",
```

Can add this back sinc eyou got the backend webpack working again with the `SKIP_PREFLIGHT_CHECK`
```suggestion
    "build:backend": "tsc -p tsconfig.backend.json && npm run build:backend:webpack",
```

---

done, thanks